influxdb3_write/src/persister.rs

```diff
@@ -120,25 +120,23 @@ impl Persister {
         &self.host_identifier_prefix
     }
 
-    /// Loads the most recently persisted catalog from object storage, and uses it to initialize
-    /// a [`LastCacheProvider`].
-    ///
-    /// This is intended to be used on server start.
-    pub async fn load_last_cache_and_catalog(
-        &self,
-        host_id: Arc<str>,
-    ) -> Result<(LastCacheProvider, Catalog)> {
-        match self.load_catalog().await? {
-            Some(c) => Ok((
-                LastCacheProvider::new_from_catalog(&c.catalog)?,
-                Catalog::from_inner(c.catalog),
-            )),
+    /// Try loading the catalog, if there is no catalog generate new
+    /// instance id and create a new catalog and persist it immediately
+    pub async fn load_or_create_catalog(&self) -> Result<Catalog> {
+        let catalog = match self.load_catalog().await? {
+            Some(c) => Catalog::from_inner(c.catalog),
             None => {
                 let uuid = Uuid::new_v4().to_string();
                 let instance_id = Arc::from(uuid.as_str());
-                Ok((LastCacheProvider::new(), Catalog::new(host_id, instance_id)))
+                info!("Created new instance id {:?}", instance_id);
```

Comment:
```suggestion
                info!(instance_id = ?instance_id, "catalog not found, creating new instance id");
```
Just encouraging use of `tracing`'s field syntax.

This is somewhat redundant with the `info!` emitted at the caller level, but I don't think it hurts to have.